forgerock-authenticator/src/main/java/org/forgerock/android/auth/DefaultStorageClient.java

```diff
@@ -173,6 +173,7 @@ public List<PushNotification> getAllNotificationsForMechanism(Mechanism mechanis
         List<PushNotification> allPushNotifications = this.getAllNotifications();
         for(PushNotification pushNotification : allPushNotifications){
             if(pushNotification.getMechanismUID().equals(mechanism.getMechanismUID())){
+                pushNotification.setPushMechanism(mechanism);
```

Comment:
for(PushNotification pushNotification : allPushNotifications) {
            if(pushNotification.getMechanismUID().equals(mechanism.getMechanismUID())) {

Give some space in the curly braces